BWEnv/src/config_manager.cc

```diff
@@ -40,12 +40,20 @@ void ConfigManager::loadDefault()
 void ConfigManager::loadGeneralSection()
 {
   port = readInt_("general", "port", 0);
+#ifdef _WIN32
   log_path = readString_("general", "log_path", "C:/tc_data/torchcraft_log_cpp_port_");
```

Comment:
Maybe %SystemDrive% instead of C: (same below)

Reply:
Forgive my ignorance, but will Windows automatically expand this to e.g. C:/, or does this require some environment variable lookup?

Reply:
I'm pretty sure the replacement is handled at the OS level. For example, this works: 
![image](https://user-images.githubusercontent.com/116363/32511766-c5fe12fa-c3c2-11e7-8acc-70313eb1c2d0.png)

Reply:
I'm not so sure -- I can do `ls ~` as well but that doesn't mean I can use it like that when opening a std::ifstream (right?) as there's no shell expansion happening. Anyway, will try it since information about this seems hard to find online.